docs/src/00-doc/04_release_management.stories.mdx

```diff
@@ -15,6 +15,10 @@ How to release a new verision is documented in the section [Developing component
 
 A version of the Design System and component library is released when the component and tokens are done and "ready" and the application wanting to use them makes use of the version in which they are contained.
 
+To release a new version run the command `lerna version [major | minor | patch | premajor | preminor | prepatch | prerelease]`. This will take of updating the version number, committing and pushing the changes.
```

Comment:
Documentation for releasing already exists in the DS docs in: https://wmde.github.io/wikit/?path=/story/documentation-how-to-create-reusable-components-developing-components--page#releasing-a-new-component

If something is required to be updated it should probably go there. Unless you are suggesting to move the instructions, in which case, the should be removed from there and added here.

Reply:
Ooh, we didn't see this existing bit of documentation. I wouldn't have expected the docs there, given most of that document is about working with the component library, while releasing concerns all (=both tokens and component lib) parts of the design system. I'd suggest to move the release instructions to this document here, because it's more centrally located.

Reply:
So the new paragraphs that were added to storybook should go in this section (README) instead? Just asking for confirmation.

Reply:
I think what @jakobw is saying is that the content from [this document](https://github.com/wmde/wikit/blob/cf58b904ed4b0b6b37f4df7bee3e5cdaf6a27225/docs/src/00-doc/03-create-reusable-components/03_developing_components.stories.mdx#releasing-a-new-component) should go in this document.

Either or, I don't hink this information should be in the README file so that we don't need to make sure that the information is synchronised between two places. We can link to here from the readme file.

Reply:
Yes, but you are saying the opposite. I can move them either way but you have to agree.

Reply:
@guergana I don't think @itamargiv and I are disagreeing. I agree that we shouldn't duplicate the same information in the README.md in the root dir of the repo. And as far as I can tell Itamar wasn't disagreeing that here would be a good place to keep the release instructions that currently reside in the "Developing Components" doc.

Reply:
Nevermind! Sorry, got confused.